cssom-view/Overview.bs

```diff
@@ -1138,18 +1137,21 @@ Note: The {{DOMRect}} object returned by {{Element/getBoundingClientRect()}} is
 
 The <dfn method for=Element caniuse=scrollintoview>scrollIntoView(<var>arg</var>)</dfn> method must run these steps:
 
-<!-- scrollIntoView() means arg=true but scrollIntoView(undefined) means arg=false, to align with impls. -->
-
 1. Let <var>options</var> be null.
-1. If <var>arg</var> is an object, let <var>options</var> be <var>arg</var>.
-1. <a lt="converted to an IDL value">Convert</a> <var>options</var> to a {{ScrollIntoViewOptions}} dictionary. [[!WEBIDL]]
-1. If <var>arg</var> is not specified or is true, let the {{ScrollIntoViewOptions/block}} dictionary member of <var>options</var> have the value "<code>start</code>", and let the {{ScrollIntoViewOptions/inline}} dictionary member of <var>options</var> have the value "<code>nearest</code>".
-1. If <var>arg</var> is false, let the {{ScrollIntoViewOptions/block}} dictionary member of <var>options</var> have the value "<code>end</code>", and let the {{ScrollIntoViewOptions/inline}} dictionary member of <var>options</var> have the value "<code>nearest</code>".
+1. If <var>arg</var> is a {{ScrollIntoViewOptions}} dictionary, set <var>options</var> to <var>arg</var>. Otherwise:
+    1. <a lt="converted to an IDL value">Convert</a> <var>options</var> to a {{ScrollIntoViewOptions}} dictionary. [[!WEBIDL]]
+    1. If <var>arg</var> is false, let the {{ScrollIntoViewOptions/block}} dictionary member of <var>options</var> to "<code>end</code>".
 1. If the element does not have any associated <a>layout box</a> terminate these steps.
 1. <a lt='scroll an element into view'>Scroll the element into view</a>
     with the options <var>options</var>.
 1. Optionally perform some other action that brings the element to the user's attention.
 
+<p class=note highlight=javascript>Passing in <code>true</code>, <code>undefined</code>, or an empty
+dictionary <code>{}</code> to {{Element/scrollIntoView()}} is equivalent to
+<code>{&nbsp;block:&nbsp;"start",&nbsp;inline:&nbsp;"nearest"&nbsp;}</code>. Passing in
+<code>false</code> is equivalent to
+<code>{&nbsp;block:&nbsp;"end",&nbsp;inline:&nbsp;"nearest"&nbsp;}</code>.
```

Comment:
I'm not sure this is needed as we don't explain all the other nuances of IDL either.